.cloudtest.yaml

```diff
@@ -12,7 +12,6 @@ import:
   - .cloudtest/kind.yaml
   - .cloudtest/vagrant.yaml
   - .cloudtest/aws.yaml
-  - .cloudtest/azure.yaml
```

Comment:
Please better change enabled=false in azure.yaml

Reply:
done

Reply:
I also suggested it at message above